deploy/kustomize/README.md

```diff
@@ -0,0 +1,116 @@
+# Deployment Guidelines
+The Hyperconverged Cluster Operator (HCO) can be deployed on an existing cluster using two methods. This first one is by editing and applying pre-built kustomization manifests; The second method is by generating required manifests using `deploy_kustomize.sh` consuming environment variables.
+
+## Method 1: Deploy with a kustomization overlay
+`manifests` folder contains pre-configured manifests ready to be deployed using kustomize, for various HCO deployment scenarios.
+The bases are:
+ 1. **marketplace** - this base is used to deploy HCO operator using CatalogSourceConfig, pointing to an OperatorSource, which makes the operator available in OLM marketplace. Thus, the resources in this base are:
+	 - CatalogSourceConfig
+	 - OperatorSource
+2. **image_registry** - this base is used to deploy HCO operator using a direct bundle image, delivered by grpc protocol. The resource this base contains is:
+	- CatalogSource
+
+Applying one of the bases alone results in HCO operator being ready for deployment manually according to [official documentation](https://docs.openshift.com/container-platform/4.3/cnv/cnv_install/installing-container-native-virtualization.html).
```

Comment:
Please avoid referring to the downstream documentation in the upstream repo.
I think we should mention this in https://github.com/kubevirt/hyperconverged-cluster-operator/blob/master/README.md instead